### PR TITLE
dts: nxp: rt1060: Fix wrong reg address of enet2

### PR DESCRIPTION
**Pull request by allemanm**
Correction of the reg address of enet2 for nxp_rt1060.

Regression introduced at 537d5c310c3e9d22f1df9a6d0bd1e250c84216fb

### CODE REVIEW

**Review by pdgendt — DISMISSED** [review]

**Review by decsny — DISMISSED** [review]

File: dts/arm/nxp/nxp_rt1060.dtsi
Comment (danieldegrasse) [inline dts/arm/nxp/nxp_rt1060.dtsi:53]:
```suggestion
			reg = <0x402d4000 0x628>;
```

This should be lowercase to keep with [devicetree styling guidelines](https://docs.kernel.org/devicetree/bindings/dts-coding-style.html):
> Hex values in properties, e.g. “reg”, shall use lowercase hex. The address part can be padded with leading zeros.

**Review by danieldegrasse — DISMISSED** [review]
Going to block, as this PR has 2 approvals and I would like to see the change made- feel free to dismiss this review on my behalf once the issue is resolved

**Review by pdgendt — APPROVED** [review]

**Review by decsny — APPROVED** [review]

**Review by danieldegrasse — APPROVED** [review]

**Review by dleach02 — APPROVED** [review]

**Pull request merged**
